cmd/node/config/external.toml

```diff
@@ -51,11 +51,13 @@
     URL = "127.0.0.1:22111"
     # After a message will be sent it will wait for an ack message if this flag is enabled
     WithAcknowledge = true
+    # The duration in seconds to wait for an acknowledgment message, after this time passes an error will be returned
+    AcknowledgeTimeoutInSec = 60
```

Comment:
[question]
What happens after an error is returned?
Do we keep the connection, drop it?

Reply:
No, in this case, the error with timeout will be returned and have to be handled.

Reply:
For this case, the outport handler will do a retry in case of an error.